python/tvm/relay/frontend/tflite.py

```diff
@@ -651,12 +701,43 @@ def convert_shape(self, op):
 
     def convert_relu(self, op):
         """Convert TFLite ReLU"""
+        try:
+            from tflite.ActivationFunctionType import ActivationFunctionType
+        except ImportError:
+            raise ImportError("The tflite package must be installed")
+
```

Comment:
I think this is unnecessary given the import of ActivationFunctionType in the constructor [here](https://github.com/apache/incubator-tvm/blob/b979bf6a7630ada055fff2d65e1cd0f8d55bb6a0/python/tvm/relay/frontend/tflite.py#L54)

Reply:
Tried this but it failed, the scope of imports is limited to the functions in which they are imported.